Signal/src/contact/OWSContactsManager.m

```diff
@@ -511,6 +556,46 @@ - (NSString *_Nonnull)displayNameForContact:(Contact *)contact
     return displayName;
 }
 
+- (NSString *_Nonnull)displayNameForSignalAccount:(SignalAccount *)signalAccount
```

Comment:
There are several methods here which seem like they could be taken out of the contact manager to make them more easily testable and to keep OWSContactsManager from continuing to endlessly bloat.

e.g. all the ones that take a SignalAccount parameter are good candidates to be moved to the SignalAccount model, or if you want a "dumber" model, to a presenter.